src/webrtc.ts

```diff
@@ -14,7 +14,7 @@ import { isFirefox } from 'browserama';
 import { definitions, Propose } from './stanza-definitions/webrtc-signaling';
 import { GenesysCloudMediaSession, IGenesysCloudMediaSessionParams, SessionEvents } from './types/media-session';
 import { isAcdJid, isScreenRecordingJid, isSoftphoneJid, isVideoJid, calculatePayloadSize, retryPromise, RetryPromise } from './utils';
-import Client from '.';
+import { Client } from './client';
```

Comment:
this was the bad import causing issues with the sendStats when using the bundled client.